Makefile.install

```diff
@@ -56,6 +56,13 @@ install-dune: $(BCONTEXT)/coq-core.install $(BCONTEXT)/coqide-server.install
 	dune install --display quiet $(DESTDIRARG) --mandir="$(MANDIR)" --prefix="$(COQPREFIX)" coq-core
 	dune install --display quiet $(DESTDIRARG) --mandir="$(MANDIR)" --prefix="$(COQPREFIX)" coqide-server
 
+# IMPORTANT NOTE: before Dune 2.9, the --docdir and --etcdir options
+# were not supported, if you are using Dune >= 2.9, you can add
+#
+# --etcdir=$(CONFIGDIR) --docdir=$(DOCDIR) , additionally, you can also add
```

Comment:
Can we do some `ifdef DUNE_29_PLUS` so that we can just say "if you are using dune >= 2.9 you can do `make DUNE_29_PLUS=1` to use docdir/etcdir", or maybe even autodetect it?

Reply:
OK